tests/cases/compiler/relativeModuleWithoutSlash.ts

```diff
@@ -11,10 +11,10 @@ export default { aIndex: 0 };
 import a from ".";
 import aIndex from "./";

```

Comment:
Could you add test for when the filename is not "/a/index.ts"? this seems to only work when your filename is "index.ts"